src/content/docs/libraries/javascript.mdx

```diff
@@ -1,20 +1,212 @@
 ---
-title: JavaScript
-description: A guide in my new Starlight docs site.
+title: TopStats JavaScript SDK
```

Comment:
I think it's better to have it remain named JavaScript, title looks better in navigation and is mostly clear.